### DISCUSSION TITLE
Depreciation after install via composer

### DISCUSSION THREAD

**Discussion by ziat1988** [sulu/sulu#6348] [General]
I run this command to install Sulu: composer create-project sulu/skeleton my-project -n
At the end, the terminal shows some depreciations:
` In Imagine.php line 194:
    Gd not installed  

  User Deprecated: The "Sulu\Bundle\CoreBundle\Controller\LocalizationController" class is deprecated since Sulu 2.0, use data from "Sulu\Bundle\AdminBundle\Controller\AdminController" instead.
 User Deprecated: Since symfony/security-http 5.1: The "Symfony\Component\Security\Http\Logout\LogoutSuccessHandlerInterface" interface is deprecated, create a listener for the "Symfony\Component\Security\Http\Event\LogoutEvent" event instead.
 User Deprecated: The "Sulu\Bundle\SecurityBundle\Security\LogoutSuccessHandler" class implements "Symfony\Component\Security\Http\Logout\LogoutSuccessHandlerInterface" that is deprecated since Symfony 5.1.
 User Deprecated: The "Sulu\Bundle\SecurityBundle\Controller\ContextsController" class is deprecated since Sulu 2.2, use data from "Sulu\Bundle\AdminBundle\Controller\AdminController" instead.
 User Deprecated: The "Sulu\Bundle\PageBundle\Repository\NodeRepository" class is deprecated since Sulu 2.0, use data from "Sulu\Component\Content\Repository\ContentRepository" instead.
 User Deprecated: The "Sulu\Bundle\PageBundle\Controller\WebspaceController" class is deprecated since Sulu 2.0, use data from "Sulu\Bundle\AdminBundle\Controller\AdminController" instead.
`

Is it normal to have that notice? Thanks

**Comment by alexander-schranz** [comment]
>  In Imagine.php line 194: Gd not installed

To run sulu you need atleast one image extension installed: `gd`, `imagick` or `vips`. Its up to you which one you want to use. Then it should also not thrown the deprecations after the install.

Sulu supports Symfony 4.4 - 5.4 at current state and so need to call deprecated Symfony methods to support this wide range supported versions. Aslong as you are not call them deprecated methods yourself you don't need to take care of them.

**Reply by ziat1988** [reply]
Thanks a lot for your help. It works perfectly
